lib/capistrano/multiconfig.rb

```diff
@@ -47,9 +47,6 @@
       load(file) if File.exists?(file)
     end
 
-    # Load SCM tasks
-    load "capistrano/#{fetch(:scm)}.rb"
```

Comment:
Shouldn't it now be `configure_scm`?
At least thats what used in https://github.com/capistrano/capistrano/blob/master/lib/capistrano/setup.rb#L30